src/compiler/typescript_to_rust/azle_generate/src/generators/vm_value_conversion/try_into_vm_value.rs

```diff
@@ -371,6 +378,12 @@ pub fn generate_try_into_vm_value_impls() -> proc_macro2::TokenStream {
             }
         }
 
+        impl CdkActTryIntoVmValue<&mut boa_engine::Context, boa_engine::JsValue> for Vec<ic_cdk::timer::TimerId> {
```

Comment:
Just remember that this will need to be changed once we merge in the list of lists code